governance/handler.go

```diff
@@ -462,3 +463,15 @@ func (gov *Governance) GetGoverningInfoAtNumber(num uint64) (bool, common.Addres
 
 	return true, govNode.(common.Address), nil
 }
+
+func (gov *Governance) GetMinimumStakingAtNumber(num uint64) (uint64, error) {
+	minStaking, err := gov.GetItemAtNumberByIntKey(num, params.MinimumStake)
```

Comment:
Previously, the naming minimumStaking was used. 
But the naming minStaking appeared in this PR. 
Does it have a different meaning?

Reply:
Nope, it is the same. Dosen't it clear `min` is an abbreviation of `minimum`?

Reply:
I think it's clear.
I just asked a question because it was different from before.
Thanks!!